src/main/scala/dpla/ingestion3/enrichments/StringEnrichments.scala

```diff
@@ -51,13 +51,20 @@ class StringEnrichments {
       rightsHolder = sourceResource.rightsHolder.map(enrichEdmAgent(_)),
       subject = sourceResource.subject.map(enrichSkosConcept(_)),
       temporal = sourceResource.temporal.map(enrichEdmTimeSpan(_)),
-      title = sourceResource.title.map(_.stripHTML.reduceWhitespace),
+      title = sourceResource.title.map(_.stripHTML
+        .reduceWhitespace
+        .cleanupLeadingPunctuation
+        .cleanupEndingPunctuation),
```

Comment:
Does it matter that `reduceWhitespace` comes before `cleanupLeadingPunctuation` and/or `cleanupEndingPunctuation`?  If so, would a comment be helpful to prevent anyone from accidentally changing the order in the future?

Reply:
I don't think the order doesn't matters here